packages/seacas/libraries/ioss/src/Ioss_ElementTopology.C

```diff
@@ -17,6 +17,18 @@
 #include <utility> // for pair
 #include <vector>  // for vector
 
+Ioss::ElementShapeMap Ioss::ElementTopology::shapeToPermutationNameMap_ = {
+  {Ioss::ElementShape::UNKNOWN, "none"},
+  {Ioss::ElementShape::POINT  , "point"},
+  {Ioss::ElementShape::LINE   , "line"},
+  {Ioss::ElementShape::TRI    , "tri"},
+  {Ioss::ElementShape::QUAD   , "quad"},
+  {Ioss::ElementShape::TET    , "tet"},
+  {Ioss::ElementShape::PYRAMID, "pyramid"},
+  {Ioss::ElementShape::WEDGE  , "wedge"},
+  {Ioss::ElementShape::HEX    , "hex"}
+};
```

Comment:
Might make sense to add a `SUPER` to the enum and a "super" here which might make for less special cases...?

Reply:
The problem with adding 'SUPER' enum and "super" to the string is that there is no way to get the specific super permutation based on the number of nodes .. that's a tricky one so I left it out. In either case, there is an override on the SuperPermutation class to return the correct 'base_topology_permutation_name()' value so I suppose that for completeness, it doesn't hurt to add the SUPER enum entry ...